### ISSUE TITLE
UItwerking Control en Interactie laag benodigd.

### ISSUE THREAD

**Issue by RvdKlip** [VNG-Realisatie/common-ground#72]
Om het commonground model te kunnen omarmen is een verdere uitwerking nodig van het 5 lagen model. Met name de Control en Interactie laag. 

Invulling van de bovenste 2 lagen raakt ook een ander punt. In de commonground vision wordt het 5 lagenmodel neergezet als 'software architecture'.  De plaats en samenhang met overige architecturen (bijvoorbeeld een Enterprise/bedrijfsarchitectuur of de gegevensarchitectuur van VNG) verdient nadere uitwerking.

**Comment by ehotting** [comment]
Correct! Het bekend geworden "5-lagen" plaatje is een vereenvoudigde weergave van de A/I laag in Archimate. Een nadere uitwerking (voor GEMMA) is in de maak - maar niet heel spannend. Qua architectuur is Common Ground niet vernieuwend, hooguit in de context van gemeenten / NL overheid. Altijd op zoek naar mensen die hier aan willen bijdragen vanzelfsprekend.

**Comment by RvdKlip** [comment]
Zeker zal/wil ik daar aan bijdragen.  Is github ook de plek om 'de nadere uitwerking voor Gemma' alvast te delen zodat bijdragen mogelijk is ? 

Blij dat mede archimate als fundering dient. Bedoel je met afkorting A/I 'Application Interface'? Archimate kent alleen een Business, Application en Technology laag.

**Comment by RvdKlip** [comment]
Leave a comment

**Comment by ehotting** [comment]
Ja, deel wat je wilt, heel graag!

A/I is mijn lokaal jargon, sorry. Application Layer dekt Applicaties en Informatie af. Bedoelde dus Application / Information, maar is inderdaad de Application Layer. Het 5-lagen model speelt zich geheel op de Application Layer af. De "control" laag gaat dus ook niet om "processen" maar over bijv. een BPMN engine wanneer iets zich als component puur in die control laag bevindt, of een app als iets zich uitspant over de bovenste twee lagen.
